src/client/routes/Settings/index.tsx

```diff
@@ -12,85 +12,72 @@ import { AlertTypes, ModalName, Theme, Language } from '@types';
 import { formatPercent, getCurrentLanguage } from '@utils';
 import { ViewContainer, ThemeBox, CustomThemeButton } from '@components/app';
 import {
-  ThemesIcon,
-  WorldIcon,
-  ClockIcon,
-  SettingsIcon,
-  Icon,
+  // ThemesIcon,
+  // WorldIcon,
+  // ClockIcon,
+  // SettingsIcon,
+  // Icon,
```

Comment:
Remove?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6